scalardb-test/src/main/java/kelpie/scalardb/Common.java

```diff
@@ -36,31 +36,26 @@ public static DistributedStorage getStorage(Config config) {
     return factory.getStorage();
   }
 
-  public static DistributedTransactionManager getTransactionManager(
-      Config config, String keyspace, String table) {
+  public static DistributedTransactionManager getTransactionManager(Config config) {
     DatabaseConfig dbConfig = getDatabaseConfig(config);
     TransactionFactory factory = new TransactionFactory(dbConfig);
-    DistributedTransactionManager manager = factory.getTransactionManager();
-    manager.with(keyspace, table);
```

Comment:
Stopped using the `DistributedTransactionManager.with()` method since it's deprecated.